package.json

```diff
@@ -36,7 +36,8 @@
     "**/js-yaml": "3.13.1",
     "**/lodash": "4.17.21",
     "**/node-gyp/tar": "2.2.2",
-    "**/prettier": "1.18.2"
+    "**/prettier": "1.18.2",
+    "**/moment": "2.29.4"
```

Comment:
Why do we need this here? I think lock file is sufficient.

Reply:
dependabot never inserts dependencies into package.json when it bumps the versions

Reply:
https://github.com/yarnpkg/yarn/issues/4986#issuecomment-395036563

* delete dependency from the lock file
* run `yarn` - it will re-add it with a newer version